### PR TITLE
fix: 3d-tiles replace refinement behaviour properly hides tile content

### PR DESCRIPTION
**Pull request by Jeremy-Gaillard**

### CODE REVIEW

File: src/Process/3dTilesProcessing.js [hunk 1]
```diff
@@ -144,8 +144,8 @@ export function process3dTilesNode(cullingTest, subdivisionTest) {
         if (node.parent.pendingSubdivision && !node.parent.additiveRefinement) {
             node.visible = false;
             // TODO: node.setDisplayed(false)?
-            if (node.material) {
-                node.material.visible = false;
+            if (node.content) {
```
Comment (peppsac) [inline src/Process/3dTilesProcessing.js]: Maybe copy-paste the commit comment here. Or maybe add a `setTileVisible()` function, including this comment.
WDYT?

Comment (Jeremy-Gaillard) [reply src/Process/3dTilesProcessing.js]: I created a setDisplayed function.

I also simplified the code a bit by removing the calls to setDisplayed when the tile was not visible.

File: src/Process/3dTilesProcessing.js [hunk 2]
```diff
@@ -138,15 +138,17 @@ export function init3dTilesLayer(context, layer) {
     });
 }
 
+function setDisplayed(node, display) {
+    if (node.content) {
```
Comment (peppsac) [inline src/Process/3dTilesProcessing.js]: You should copy-paste a comment here explaining why your not simply doing `node.material.visible = false`

Comment (Jeremy-Gaillard) [reply src/Process/3dTilesProcessing.js]: Done

**Review by peppsac — APPROVED** [review]

**Pull request merged**